### PR TITLE
Prkrishn/release/2.1

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: testassets/SimpleRepo/korebuild.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/release/2.1/tools/korebuild.schema.json",
-  "channel": "release/2.1"
+  "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/dev/tools/korebuild.schema.json",
+  "channel": "dev"
```
Comment (pranavkm) [inline testassets/SimpleRepo/korebuild.json]: Fix

File: version.props [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
-    <KoreBuildChannel>release/2.1</KoreBuildChannel>
+    <KoreBuildChannel>dev</KoreBuildChannel>
```
Comment (natemcmaster) [inline version.props]: Fix this too.

**Comment by pranavkm** [comment]
🆙 📅

File: korebuild.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
 {
   "$schema": "./tools/korebuild.schema.json",
-  "channel": "release/2.1"
+  "channel": "dev"
```
Comment (natemcmaster) [inline korebuild.json]: acutally, this should be changed too. It's used by test.ps/sh

Comment (pranavkm) [reply korebuild.json]: Done.

**Pull request merged**